tests/test_alias.rs

```diff
@@ -131,7 +131,7 @@ fn test_alias_cannot_override_builtin() {
     // Alias should be ignored
     let stdout = test_env.jj_cmd_success(&repo_path, &["log", "-r", "root"]);
     insta::assert_snapshot!(stdout, @r###"
-    o 000000000000  1970-01-01 00:00:00.000 +00:00 000000000000
+    o 0[000000000]  1970-01-01 00:00:00.000 +00:00 0[000000000]
```

Comment:
Thanks for fixing this, I just ran into it. (No action needed)

Reply:
Curiosity: Doing `jj up 0000000` in my main repo worked, but put `jj` in a weird state where it kept freezing up. There's a bug there somewhere (maybe just for colocated repos). Don't try this on your main repo.